colrev/ops/load_utils_nbib.py

```diff
@@ -146,34 +126,38 @@ def load_nbib_entries(self) -> dict:
 
         records = {}
         for ind, record in enumerate(records_list):
-            record[Fields.ID] = str(ind).rjust(6, "0")
+            record[Fields.ID] = str(ind + 1).rjust(6, "0")
             records[record[Fields.ID]] = record
-
         return records
 
-    def convert_to_records(self, *, entries: dict) -> dict:
+    def convert_to_records(self, *, entries: dict, mapping: dict) -> dict:
```

Comment:
Could you refactor this part in a similar way to the `load_utils_ris.py` (the `apply_entrytype_mapping` and `map_keys` methods)?

Reply:
Sure